src/fidesops/service/masking/strategy/masking_strategy_hash.py

```diff
@@ -15,6 +15,7 @@
 
 
 HASH = "hash"
+SUPPORTED_DATA_TYPES = ["string"]
```

Comment:
If we're going to have `Supported Types`, it should probably be an abstract method or maybe a default implementation  on the parent MaskingStrategy class. I don't think this should be a module level variable.

Also, since we're repeatedly calling "x in SUPPORTED_DATA_TYPES" , making SUPPORTED_DATA_TYPES a set (since we're only creating it once) will be faster.

Reply:
I'm using the abstract method `data_type_supported` already, it's just the `SUPPORTED_DATA_TYPES` const that's specific to each module. I think a `data_type_supported` abstract method works better than a `supported_data_types` abstract method bc it leaves the specific details up to the module. i.e. for null masking, we always support the data type, so the caller of a `supported_data_types` abstract method would need to have a special case for null masking

Reply:
Makes sense, I'll update!